INFO:root:
Selector: High Level Ordering of Strategies
| Sequence: Offensive Strategy
| | Check: have_largest_fleet
| | Action: attack_weakest_enemy_planet
| Sequence: Spread Strategy
| | Check: if_neutral_planet_available
| | Action: spread_to_weakest_neutral_planet
| Action: attack_weakest_enemy_planet

DEBUG:root:Executing:Selector: High Level Ordering of Strategies
DEBUG:root:Executing:Sequence: Offensive Strategy
DEBUG:root:Executing:Check: have_largest_fleet
DEBUG:root:Result: Check: have_largest_fleet -> Failure
DEBUG:root:Result: Sequence: Offensive Strategy -> Failure
DEBUG:root:Executing:Sequence: Spread Strategy
DEBUG:root:Executing:Check: if_neutral_planet_available
DEBUG:root:Result: Check: if_neutral_planet_available -> Success
DEBUG:root:Executing:Action: spread_to_weakest_neutral_planet
DEBUG:root:Order:1 13 60.0
DEBUG:root:Result: Action: spread_to_weakest_neutral_planet -> Success
DEBUG:root:Result: Sequence: Spread Strategy -> Success
DEBUG:root:Result: Selector: High Level Ordering of Strategies -> Success
DEBUG:root:Finish turn

DEBUG:root:Executing:Selector: High Level Ordering of Strategies
DEBUG:root:Executing:Sequence: Offensive Strategy
DEBUG:root:Executing:Check: have_largest_fleet
DEBUG:root:Result: Check: have_largest_fleet -> Failure
DEBUG:root:Result: Sequence: Offensive Strategy -> Failure
DEBUG:root:Executing:Sequence: Spread Strategy
DEBUG:root:Executing:Check: if_neutral_planet_available
DEBUG:root:Result: Check: if_neutral_planet_available -> Success
DEBUG:root:Executing:Action: spread_to_weakest_neutral_planet
DEBUG:root:Result: Action: spread_to_weakest_neutral_planet -> Failure
DEBUG:root:Result: Sequence: Spread Strategy -> Failure
DEBUG:root:Executing:Action: attack_weakest_enemy_planet
ERROR:root:Error in bot.
Traceback (most recent call last):
  File "behavior_tree_bot/bt_bot.py", line 57, in <module>
    do_turn(planet_wars)
  File "behavior_tree_bot/bt_bot.py", line 45, in do_turn
    behavior_tree.execute(planet_wars)
  File "/Users/naummarkenzon/Desktop/UCSC-Comp-Sci-Projects/CMPM146/cmpm146-P4/behavior_tree_bot/bt_nodes.py", line 8, in logged_fn
    result = fn(self, state)
  File "/Users/naummarkenzon/Desktop/UCSC-Comp-Sci-Projects/CMPM146/cmpm146-P4/behavior_tree_bot/bt_nodes.py", line 52, in execute
    success = child_node.execute(state)
  File "/Users/naummarkenzon/Desktop/UCSC-Comp-Sci-Projects/CMPM146/cmpm146-P4/behavior_tree_bot/bt_nodes.py", line 8, in logged_fn
    result = fn(self, state)
  File "/Users/naummarkenzon/Desktop/UCSC-Comp-Sci-Projects/CMPM146/cmpm146-P4/behavior_tree_bot/bt_nodes.py", line 89, in execute
    return self.action_function(state)
  File "/Users/naummarkenzon/Desktop/UCSC-Comp-Sci-Projects/CMPM146/cmpm146-P4/behavior_tree_bot/behaviors.py", line 37, in attack_weakest_enemy_planet
    planets.remove(strongest_planet)
ValueError: list.remove(x): x not in list
